app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.split()[1] != command.split()[2]:
```

Comment:
Move command.split() to a variable
Validate variable length
Validate that first argument is exactly "cp" command
And use unpacking instead of index access